charmhelpers/contrib/openstack/context.py

```diff
@@ -554,6 +554,10 @@ def _resolve(key):
                             ctxt, keystonemiddleware_os_release)
 
                 if self.context_complete(ctxt):
+
+                    ctxt['audit_middleware'] = \
+                        False if config('audit-middleware') is None else config('audit-middleware')
```

Comment:
Minor suggestion, but his tends to be simpler:

```python
ctxt['audit_middleware] = config('audit-middleware') or False
```

Reply:
updated